lib/tmdb_handler.rb

```diff
@@ -37,11 +39,9 @@ def tmdb_handler_person_autocomplete(query)
   end
 
   def tmdb_handler_movie_more(id)
-    @movie_url = "#{BASE_URL}/movie/#{id}?api_key=#{ENV['tmdb_api_key']}&append_to_response=trailers,credits,releases"
-    @result = JSON.parse(open(@movie_url).read, symbolize_names: true)
-    @movie = MovieMore.parse_result(@result)
-
-    @production_companies = @result[:production_companies]
+    movie_url = "#{BASE_URL}/movie/#{id}?api_key=#{ENV['tmdb_api_key']}&append_to_response=trailers,credits,releases"
+    result = JSON.parse(open(movie_url).read, symbolize_names: true)
+    MovieMore.parse_result(result)
```

Comment:
The `tmdb_handler_movie_more` method is the centerpiece of this PR. I've removed 4 instance variables and now this method returns a single `MovieMore` object which gets assigned to `@movie` (in controllers) or `movie` in the other methods that call it.